bin/hdfs-mesos

```diff
@@ -4,4 +4,4 @@ if [ -z "$PORT0" ] ; then
   PORT0="8765"
 fi
 
-exec java -cp lib/*.jar -Dmesos.conf.path=etc/hadoop/mesos-site.xml -Dmesos.hdfs.config.server.port=$PORT0 org.apache.mesos.hdfs.Main
+exec `if [ -z "$JAVA_HOME" ]; then echo java; else echo $JAVA_HOME/bin/java; fi` -cp lib/*.jar -Dmesos.conf.path=etc/hadoop/mesos-site.xml -Dmesos.hdfs.config.server.port=$PORT0 org.apache.mesos.hdfs.Main
```

Comment:
Please separate this if else into a separate statement to avoid too much nesting and better readability